LDAR-Sim_inputs.md

```diff
@@ -897,6 +933,105 @@ See Section 3.12.
 
 See Section 2.11.
 
+# Scheduling Inputs
+
+## route\_planning
+
+**Data type:** Boolean
+
+**Default input:** N/A
+
+**Description:** A binary True/False to activate the route planning. Route planning allows LDAR crew to choose the nearest facility and home bases to visit based on the shortest travelling cost. The travelling cost is travel time that is calculated using the Haversine distance metric and sampled maximum speed limit of travelling. This will be improved in the future, especially for OGI and trucks.  
+
+**Notes on acquisition:** It requires user to also define input for home_bases, Speed_list and, LDAR_crew_init_location. 
+
+**Notes of caution:** Only aircraft, truck, and OGI methods can use this functionality.
+
+## geography
+
+**Data type:** Boolean
+
+**Default input:** N/A
+
+**Description:** A binary True/False to activate the using real geography coordinates for home bases. The geography functionality allows LDAR crew to depart from the home base (town or city, or airport) at the start of each day and return to the home base at the end of each day.  
+
+**Notes on acquisition:** It requires user to also define input for home_bases, Speed_list and, LDAR_crew_init_location. 
+
+**Notes of caution:** Only aircraft, truck, and OGI methods can use this functionality.
+
+## home\_bases
+
+**Data type:** Character string that specifies the name of the csv file that contains all of the required data on the home bases that used for LDAR scheduling.
+
+**Default input:** N/A
+
+**Description:** At a bare minimum, the csv must contain the following columns: 'lat', 'lon’, where ‘lat’ and ‘lon’ are coordinates of each home base.   
+
+**Notes on acquisition:** It is only required if route_planning or geography is activated.   
+
+**Notes of caution:** Only aircraft, truck, and OGI methods can use this functionality.
+
+## speed\_list
+
+**Data type:** list
+
+**Default input:** [60.0,70.0,80.0,90.0] for OGI and truck methods, [200.0,210.0,220.0,230.0] for aircraft method. 
+
+**Description:** A list of speed limits that define the maximum travelling speed of technologies. A random speed is sampled from this list when calculating the travel time between two facilities or between the facility and home base. Can also be a list with a single value.
```

Comment:
For now, it assumes crews travel at a constant speed for all mobile methods. the speed of travelling is sampled from the speed list. This will be removed in the future once we find a way to extract route/driving speed from real road network.